lib/api/calendars.ts

```diff
@@ -1,10 +1,32 @@
+import convert from 'xml-js';
 import { DAVClient } from '../DAVClient';
 
 const BASE_PATH = '/calendars';
 
-export const getInbox = (client: DAVClient) => async (userId: string): Promise<string> => {
-  return client.requestText({
-    url: `${BASE_PATH}/${userId}/inbox`,
+export interface MultiStatusResponse {
```

Comment:
This doesn't need to be exported.

We may put XML interfaces in a separate file